src/api/OnmsServer.ts

```diff
@@ -82,6 +82,18 @@ export class OnmsServer {
     return uri.toString();
   }
 
+  /**
+   * Whether this server object is the same as another.
```

Comment:
I think that is obvious (-:

Reply:
Yeah, but the doc plugin requires 100% coverage on public methods.  ;)  I'll change it to something a bit more useful.